### PR TITLE
Fixed SVG Background Maker keyboard issue

### PR DESCRIPTION
**Pull request by sarvaje**

### CODE REVIEW

File: svgradientbackgroundmaker/styles/svgdemo.css [hunk 1]
```diff
@@ -102,16 +102,18 @@ fieldset {
 .positionThumbnail,
 .sizeThumbnail {
 	display: inline-block;
-	float: left;
+	float:left;
+	font-family: 'Montserrat', 'Segoe UI', sans-serif;
+	box-sizing: content-box;
 	font-size: 1.1rem;
 	width: 10.5rem;
 	height: 1.8rem;
```
Comment (alrra) [inline svgradientbackgroundmaker/styles/svgdemo.css]: Maybe this should be increased a bit?

File: svgradientbackgroundmaker/styles/svgdemo.css [hunk 2]
```diff
@@ -102,16 +102,18 @@ fieldset {
 .positionThumbnail,
 .sizeThumbnail {
 	display: inline-block;
-	float: left;
+	float:left;
```
Comment (alrra) [inline svgradientbackgroundmaker/styles/svgdemo.css]:
```css
float:<space>left;
```

**Review by alrra — APPROVED** [review]

File: svgradientbackgroundmaker/styles/svgdemo.css (hunk 1 above)
Comment (sarvaje) [reply svgradientbackgroundmaker/styles/svgdemo.css]: I fix it removing the fixed height.

Comment (alrra) [reply svgradientbackgroundmaker/styles/svgdemo.css]: > I fix it removing the fixed height.

👍 

---

* Edge:

   

* Chrome

  

* Firefox:

**Comment by melanierichards** [comment]
@sarvaje @dstorey going to merge this so I can make other changes to the markup!

**Pull request merged**